docs/scripts/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
```

Comment:
something for later, but there is probably a github action for this instead of having to do it via scripts. I dont think the scripts should be availabel tbh, there should eb no need for someone to publish the docs manually, ever.

Reply:
errrh, this was meant for the publish.sh script, not the build ¬_¬